redis.go

```diff
@@ -168,15 +168,24 @@ func (s *RedisServer) Run(ctx context.Context) error {
 
 	// now start request/response workers and proxy calls and responses
 	pullArgs = append(pullArgs, redisPullTimeout) //the pull timeout
-	ch := s.Start(ctx, s.workers, s.cb)
+	workerCtx, shutdown := context.WithCancel(context.Background())
```

Comment:
So I get the problem right. The context here cannot be the same that the one we get in the Run method ?

Reply:
Yes, but as a user of zbus u don't have to worry about it. I create a new context, and cancel it when the passed context is cancelled, but I make sure that there is no redis poll anymore to not lose jobs